### ISSUE TITLE
[ QUESTION ] could libexif be a source for image a11y checks?

### ISSUE THREAD

**Issue by OlgaTCK** [libexif/libexif#27]
I am wondering if parsing EXIFs of images from libexif could also be a source for image a11y checks like description tags aka "Alt"-descriptions. 
Would be interesting to know your opinion.

I've got an error while first run of application which uses libexif. 
[debug.log](https://github.com/libexif/libexif/files/3854641/debug.log)
The application checks accessibility issues on websites: https://github.com/open-indy/Koa11y/issues/55

**Comment by ndim** [comment]
FTR: Apparently, "a11y" stands for "accessibility".

(Remark from someone only familiar with i18n and l10n so far.)

**Comment by OlgaTCK** [comment]
Of course,  "a11y" meaning "accessibility". I try understand if the error described in the debug log file means that the app using libexif will be unable to access image description tags, for example.
Is it possible, @ndim?

**Comment by dfandrich** [comment]
libexif is able to extract EXIF tags on images, so if part of accessibility testing involves looking at embedded image metadata like description tags, then it can be used. The error message you mention comes from the application so you'll have to ask the authors what it means.

**Comment by OlgaTCK** [comment]
Thanks a lot for the details, @dfandrich. I already asked the authors and they say, the parts of the libexif were turned off on some purpose but it shouldn't affect the application during a11y checking.

**Comment by dfandrich** [comment]
In any case, I don't see anything we can help you with here.
